### PR TITLE
Micro LazyLoader now has a getter for the definition property #11940

### PR DESCRIPTION
**Pull request by VeryStrongFingers**
As per issue #11940 I have added a getter to get defintiion, for the LazyLoader (Phalcon\Mvc\Micro\LazyLoader) class

### CODE REVIEW

File: phalcon/mvc/micro/lazyloader.zep [hunk 1]
```diff
@@ -62,4 +62,9 @@ class LazyLoader
 		 */
 		return call_user_func_array([handler, method], arguments);
 	}
+
+	public function getDefinition()
+	{
+		return this->_definition;
+	}
```
Comment (sergeyklay) [inline phalcon/mvc/micro/lazyloader.zep]: What about:
```zephir
protected _definition { get };
```

Comment (Mechzeit) [inline phalcon/mvc/micro/lazyloader.zep]: @sergeyklay can you link me to relevant documentation about that? I'm new to zephir :^)

Comment (Jurigag) [inline phalcon/mvc/micro/lazyloader.zep]: https://docs.zephir-lang.com/en/latest/oop.html#getter-setter-shortcuts

**Comment by sergeyklay** [comment]
@Mechzeit

**Comment by Mechzeit** [comment]
New PR: #11951

**Pull request closed without merging**